src/bitcoin-cli-res.rc

```diff
@@ -17,13 +17,13 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoin-cli (OSS RPC client for Bitcoin)"
+            VALUE "FileDescription",    "bitcoin-cli (JSON-RPC client for Bitcoin Core)"
             VALUE "FileVersion",        VER_FILEVERSION_STR
             VALUE "InternalName",       "bitcoin-cli"
             VALUE "LegalCopyright",     COPYRIGHT_STR
             VALUE "LegalTrademarks1",   "Distributed under the MIT software license, see the accompanying file COPYING or http://www.opensource.org/licenses/mit-license.php."
             VALUE "OriginalFilename",   "bitcoin-cli.exe"
-            VALUE "ProductName",        "Bitcoin-cli"
+            VALUE "ProductName",        "bitcoin-cli"
```

Comment:
Also: you're not making the same first letter decapitalization change in `src/bitcoind-res.rc` and `src/qt/rec/bitcoin-qt-res.rc`